### ISSUE TITLE
Clarification of project license - LGPL is not compatatible with Apache License 2.0

### ISSUE THREAD

**Issue by justinmclean** [unitycatalog/unitycatalog#75]
**Describe the bug**

The NOTICE includes code licensed under the LGPL, which is not compatible with the Apache license. Is the use of Hibernate optional? The ASF regards the LGPL as a "Category X" license—see [1]. Category X basically means it can't be included in a distribution or be a dependency.

1. https://www.apache.org/legal/resolved.html#category-x

**To Reproduce**

See the LGPL license listed in the NOTICE file.

**Expected behavior**

Only dependencies or bundled software compatible with the Apache license should be listed.

Also, as an aside, the NOTICE file is generally not the place to list licenses; they should be listed in LICENSE.

**System [please complete the following information]:**

N/A

**Additional context**

N/A

**Comment by tdas** [comment]
Hi @justinmclean thank you for raising this please help me understand this. This project is Apache 2.0 Licensed, but not part of the Apache Software Foundation. So does the restriction to use LGPL still apply?

**Comment by justinmclean** [comment]
Yes it does, as the LGPL license is not compatible with the ALv2 (although the reverse is not true) see https://www.apache.org/licenses/GPL-compatibility.html
